src/Streamly/Internal/Data/Stream/IsStream/Transform.hs

```diff
@@ -566,13 +566,31 @@ trace_ eff = Serial.mapM (\x -> eff >> return x)
 
 -- | Scan a stream using the given monadic fold.
 --
+-- Non-terminating folds can be terminated by a scan.  For example,
```

Comment:
I don't understand this statement. I can't see how the example below justifies this statement.

Reply:
I will remove it.